spring-cloud-dataflow-rest-resource/src/main/java/org/springframework/cloud/dataflow/rest/resource/StreamStatusResource.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.rest.resource;
+
+import org.springframework.hateoas.CollectionModel;
+import org.springframework.hateoas.PagedModel;
+import org.springframework.hateoas.RepresentationModel;
+
+/**
+ * @author Christian Tzolov
+ */
+public class StreamStatusResource extends RepresentationModel<StreamStatusResource> {
+
+	private String name;
+
+	private CollectionModel<AppStatusResource> applications;
+
+	public String getName() {
+		return name;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	public String getVersion() {
+		try {
+			return applications.iterator().next()
```

Comment:
I hope we won't get into NPE here. Is there a way to simplify this logic? Also, please try to move these keys (skipper.release.version etc.,) into constants and refer them

Reply:
sure i will add some checks to prevent NPE response and will factor out the constant.